lib/dev/ui/spinner.rb

```diff
@@ -91,6 +103,10 @@ def inset
           def inset_width
             @inset_width ||= Dev::UI::ANSI.printing_width(inset)
           end
+
+          def right_pad
+            ' ' * @longest_line_width
```

Comment:
I know this is not fool proof and could under-pad, but as a default I think it's good enough.